.github/workflows/run_doublet-detection.yml

```diff
@@ -43,11 +43,17 @@ jobs:
           conda activate openscpca-doublet-detection
           ./download-data.py --test-data --format SCE
 
-      - name: Run doublet-detection module
+      - name: Run doublet-detection module on benchmark test data
+        run: |
+          cd ${MODULE_PATH}
+          conda activate openscpca-doublet-detection
```

Comment:
```suggestion
```
I don't think you need this since the environment was activated in the download test data step.

Reply:
let's see!